### ISSUE TITLE
Make the app data-driven

### ISSUE THREAD

**Issue by joshJarr** [tomwieck/triffid#7]

**Comment by tomwieck** [comment]
Took a quick look at this, but still getting the same CORS error. I've had a look at the documentation and I can't see any way of adding headers to the backend https://pico.readthedocs.io/en/latest/index.html# 

If anyone has any ideas that'd be great, if not it might be worth looking into a bit more of a fleshed out backend framework (Django?)

**Comment by fznsakib** [comment]
Yeah I think I'll implement Flask instead, it's better documented and I've used it a bit before. Not sure as to how I can test from front-end, so I'll make the change from Pico to Flask and merge it and you guys can have a look if it works or not

**Comment by jalal** [comment]
Seems to work for me here. 
I've got some opinions on the Rest API endpoints, but I'll put something on Slack about that.

**Comment by tomwieck** [comment]
Started this with the Park List, going to mark this as 'complete' and then we can make more fine grained tasks for each section (making the map data driven etc.)
